### ISSUE TITLE
For each Audio Page/Subpage be able to switch between Browse View and Archive View

### ISSUE THREAD

**Issue by abm-avmonk** [abhayagiri/abhayagiri-website#22]
For a number of the pages, it would be nice to have a much more condensed version of all the talks which would make it easier to download all of the talks from a given Ajahn or topic. The idea would be that the normal "Browse View" would be nice for interacting with on your phone, or if you aren't too particular about the talk you want, but the "Archive View" would be more for the enthusiastic person who would like to download multiple talks on a specific topic, or all the talks by a given Ajahn, or all of the talks for a retreat. It would allow them to readily see all the talks available and easily access very old talks if they have listened to all the recent ones. 

The pages that I imagined having this option are:
Dhamma talks: each Ajahn Subpage
Dhamma talks: each Topic Subpage
Retreats and Collections
Morning Reflections
(Chanting will probably need to be structured a little differently than the others anyway)

The default would be exactly as we have it currently. However somewhere visible, maybe prominently below the picture on the left or on an extra toolbar at the top, Have a link that says, "Switch to Archive View." 

The Archive view would be very similar to what I described earlier in the comment:
"Update for Audio Page(s) to make Browsing More available"

**For each Ajahn Subpage** 
We would have a list view of all the dhamma talks available there organized by year. So, for Ajahn Amaro it would look like:

2017 Anicca and Worldly life (Play) (Download)
2014 Emptiness (Play) (Download)
         Cultivating Metta (Play) (Download)
         etc
2013 Mara's Temptations (Play) (Download)
         Gratitude for our Parents (Play) (Download)
         ...
2008

I'm thinking it would looks something similar to Ajahn Geoff's setup for his evening dhamma talks:
http://www.dhammatalks.org/mp3_index.html

Ideally, if you clicked on the talk it would expand so that you could see the description of the talk right there in the page. If that is too complicated we can just include a link that sends the use to that talk's page with it's description. 

**For each Topic Subpage**
The topic Subpages are a little different because we can sort them by Ajahn or by date published. I think it makes sense to have the default to be sort by date. If we can also sort by Ajahn, that would be wonderful. The main difference from the Ajahn Subpage would be that it would have the Ajahn in the description. so we would have:

Anicca/Impermanence Subpage
2017 Anicca and Worldly Life by Ajahn Amaro (Play) (Download)
2014 Impermanence and Emptiness by Ajahn Pasanno (Play) (Download)
         Coping with Anicca by Ajahn Jotipalo (Play) (Download)
         etc
2013 Delusion and Persistence Ajahn Karunadhammo (Play) (Download)
         How to integrate Anicca Moment by Moment by Tan Kaccana (Play) (Download)
         ...
2008

If we could also have a "Sort by Ajahn" button, I would imagine it to be 

Anicca/Impermanence Subpage
Ajahn Amaro 
    Anicca and Worldly Life (Play) (Download)
Ajahn Jotipalo
    Coping with Anicca (Play) (Download)
Ajahn Karunadhammo 
    Delusion and Persistence (Play) (Download)
 Ajahn Pasanno
    Impermanence and Emptiness (Play) (Download)
    Anicca in Every Moment (Play) (Download)
etc....

**Comment by abm-webmonk** [comment]
I don't think this issue is too urgent. Also, per in house discussion, it seems this might take a lot of work to implement. Thus, I am doubting whether it is worth that work.

**Comment by abm-webmonk** [comment]
This is what I was thinking of as the Archive View. The Browse view would be exactly the same as the current subpages of the talks aside from having the "Browse View | Archive View" label below the banner picture. 

![Archive View](https://user-images.githubusercontent.com/38106637/79050739-dda9b980-7be0-11ea-81bb-aa72ea433600.png)

**Comment by abm-webmonk** [comment]
Also, this would be the Browse View. 

![Browse View](https://user-images.githubusercontent.com/38106637/79050802-4bee7c00-7be1-11ea-8b0e-c746f28ea752.png)
